resources/lib/PayoneApi/Request/Authorization/AmazonPay.php

```diff
@@ -8,18 +8,34 @@
 use PayoneApi\Request\Parts\RedirectUrls;
 
 /**
- * Class PayPal
+ * Class AmazonPay
  */
 class AmazonPay extends AuthorizationRequestAbstract
 {
+    /** @var string */
     const WALLET_TYPE = 'AMZ';
 
+    /** @var string */
     protected $clearingtype = ClearingTypes::WALLET;
-    /**
-     * @var string
-     */
+
+    /** @var string */
     private $wallettype = self::WALLET_TYPE;
 
+    /** @var string */
+    private $workorderid;
```

Comment:
camelCase

Reply:
Wird über json_serialize in den fertigen Request gewandelt. Die abweichenden Variablennamen sind die Parameter im Request später.